### ISSUE TITLE
ANR

### ISSUE THREAD

**Issue by William353** [firebase/quickstart-android#1274]
### Step 1: Describe your environment

  * Android device: too manay 
  * Android OS version: all  5、6、7、8、9、10 
  * Google Play Services version: 17.0.0
  * Firebase/Play Services SDK version: 
    implementation "com.google.firebase:firebase-core:17.4.4"
    implementation "com.google.firebase:firebase-messaging:21.0.0"
    implementation "com.google.firebase:firebase-config:18.0.0"
  
### Step 2: Describe the problem:
Getting too many ANR by com.google.firebase.iid.FirebaseInstanceIdReceiver in Google play console ANR & Crash logs.

Broadcast of Intent { act=com.google.android.c2dm.intent.RECEIVE flg=0x11000010 pkg=packename cmp=packename/com.google.firebase.iid.FirebaseInstanceIdReceiver (has extras) }

Basically all anr caused by push. I think it was caused by launching the app when the push arrived, is it possible to receive pushes in the child process?

**Comment by samtstern** [comment]
@William353 you need to make sure that your app does not do too much heavy work after receiving this broadcast. I am going to close this because it's not a new issue or an issue with this quickstart, please read these threads for more information:
https://github.com/firebase/quickstart-android/issues?q=is%3Aissue+ANR
